packages/dialect-react/utils/errors.ts

```diff
@@ -49,6 +51,20 @@ export const noAccount: ParsedErrorData = {
   matchers: ['Account does not exist'],
 };
 
+export const incorrectEmail: ParsedErrorData = {
```

Comment:
Since email has web2 communications, I don't think we need to handle it through matchers, it must have a normalized response, since we control the backend. wdyt?

Reply:
good suggestion

Reply:
added a TODO comment for now, should resolve later on